android/app/src/main/java/org/cryptimeleon/incentive/app/settings/SettingsViewModel.kt

```diff
@@ -25,4 +25,8 @@ class SettingsViewModel @Inject constructor(
     val providerPublicKey = cryptoRepository.observeCryptoMaterial().asLiveData().map {
         it!!.ppk.toString()
```

Comment:
What does !! do?

Reply:
https://kotlinlang.org/docs/null-safety.html#the-operator
Basically throws a null pointer exception if it is null. Kotlin requires the user to explicitly state this or handle the null case for nullable fields.